components/authz-service/storage/postgres/migration/legacy/iam_migration_test.go

```diff
@@ -268,6 +275,13 @@ func queryTestPolicy(ctx context.Context, id string, db *sql.DB) (*v2Policy, err
 
 	resp, err := queryPolicy(ctx, id, tx)
 	if err != nil {
+		// The not found on json unmarshall case
+		if strings.HasPrefix(err.Error(), "sql: Scan error on column index 0") &&
```

Comment:
If a policy wasn't found, we got this garbled pq error. This just gives a future test-breaker an easier glimpse at the problem.